describe.skip does not skip assertions in block

I expect that skipped `describe` blocks don't run any code inside, and especially not assertions.

**Steps to Reproduce**

1. Create `denobug.test.ts`
```ts
import { describe, it } from '@std/testing/bdd';
import { assertExists } from 'jsr:@std/assert';

let skipTestingThisVariable: string;

describe.skip('skip everything in here', () => {
  assertExists(skipTestingThisVariable);  // <-- this line should be skipped
  it('should work', () => {
    assertExists(true);
  });
});
```
2. Run `deno test denobug.test.ts`

```
Uncaught error from ./denobug.test.ts FAILED

 ERRORS 

./denobug.test.ts (uncaught error)
error: (in promise) AssertionError: Expected actual: "undefined" to not be null or undefined.
    throw new AssertionError(msg);
          ^
    at assertExists (https://jsr.io/@std/assert/1.0.8/exists.ts:29:11)
    at file:///~/deno/denobug.test.ts:7:3
    at new TestSuiteInternal (https://jsr.io/@std/testing/1.0.5/_test_suite.ts:91:23)
    at describe (https://jsr.io/@std/testing/1.0.5/bdd.ts:1111:22)
    at Function.describeIgnore [as ignore] (https://jsr.io/@std/testing/1.0.5/bdd.ts:1176:10)
    at Function.describeSkip [as skip] (https://jsr.io/@std/testing/1.0.5/bdd.ts:1208:19)
    at file:///~/deno/denobug.test.ts:6:10
This error was not caught from a test and caused the test runner to fail on the referenced module.
It most likely originated from a dangling promise, event/timeout handler or top-level code.

 FAILURES 

./denobug.test.ts (uncaught error)

FAILED | 0 passed | 1 failed (0ms)

error: Test failed
```

**Expected behavior**

The test should be ignored. If the `assertExists(skipTestingThisVariable);` is moved into the `it`, then it's correctly ignored

```ts
import { describe, it } from '@std/testing/bdd';
import { assertExists } from 'jsr:@std/assert';

let skipTestingThisVariable: string;

describe.skip('skip everything in here', () => {
  it('should work', () => {
    assertExists(skipTestingThisVariable);  // test is ignored now
    assertExists(true);
  });
});
```

**Environment**

- deno version: 2.1.2
- std version: "jsr:@std/expect@*": "1.0.8"

--- a/testing/_test_suite.ts
+++ b/testing/_test_suite.ts
@@ -82,7 +82,40 @@ export class TestSuiteInternal<T> implements TestSuite<T> {
     this.symbol = Symbol();
     TestSuiteInternal.suites.set(this.symbol, this);
 
-    const { fn } = describe;
+    const { fn, ignore } = describe;
+    if (ignore) {
+      const {
+        name,
+        only,
+        permissions,
+        sanitizeExit,
+        sanitizeOps,
+        sanitizeResources,
+      } = describe;
+      const options: Deno.TestDefinition = {
+        name,
+        fn: async () => {},
+        ignore: true,
+      };
+      if (only !== undefined) {
+        options.only = only;
+      }
+      if (permissions !== undefined) {
+        options.permissions = permissions;
+      }
+      if (sanitizeExit !== undefined) {
+        options.sanitizeExit = sanitizeExit;
+      }
+      if (sanitizeOps !== undefined) {
+        options.sanitizeOps = sanitizeOps;
+      }
+      if (sanitizeResources !== undefined) {
+        options.sanitizeResources = sanitizeResources;
+      }
+      TestSuiteInternal.registerTest(options);
+      return;
+    }
+
     if (fn) {
       const temp = TestSuiteInternal.current;
       TestSuiteInternal.current = this;

--- a/testing/bdd_test.ts
+++ b/testing/bdd_test.ts
@@ -1122,6 +1122,45 @@ Deno.test("describe()", async (t) => {
 
   await t.step("ignore", async (t) => {
     /**
+     * Asserts that `Deno.test` is called with the correct options for the `describe` call in the callback function.
+     * This is used to reduce code duplication when testing calling `describe` with different call signatures.
+     */
+    async function assertIgnoreOptions(
+      expectedOptions: Omit<Deno.TestDefinition, "name" | "fn">,
+      cb: (fns: readonly [Spy, Spy]) => void,
+    ) {
+      using test = stub(Deno, "test");
+      const fns = [spy(), spy()] as const;
+      try {
+        cb(fns);
+        assertSpyCall(test, 0);
+        const call = test.calls[0];
+        const options = call?.args[0] as Deno.TestDefinition;
+        assertEquals(
+          Object.keys(options).sort(),
+          ["name", "fn", ...Object.keys(expectedOptions)].sort(),
+        );
+        assertObjectMatch(options, {
+          name: "example",
+          ...expectedOptions,
+        });
+
+        assertSpyCalls(fns[0], 0);
+        assertSpyCalls(fns[1], 0);
+
+        const context = new TestContext("example");
+        const result = options.fn(context);
+        assertStrictEquals(Promise.resolve(result), result);
+        assertEquals(await result, undefined);
+        assertSpyCalls(context.spies.step, 0);
+
+        assertSpyCalls(fns[0], 0);
+        assertSpyCalls(fns[1], 0);
+      } finally {
+        TestSuiteInternal.reset();
+      }
+    }
+    /**
      * Asserts that `Deno.test` is called with just the name, ignore, and function for the `describe.ignore` call in the callback function.
      * In addition to that, it asserts that the individual test cases registered with `it` use the test step API correctly.
      * This is used to reduce code duplication when testing calling `describe.ignore` with different call signatures.
@@ -1129,7 +1168,7 @@ Deno.test("describe()", async (t) => {
     async function assertMinimumOptions(
       cb: (fns: readonly [Spy, Spy]) => void,
     ) {
-      await assertOptions({ ignore: true }, cb);
+      await assertIgnoreOptions({ ignore: true }, cb);
     }
 
     /**
@@ -1140,7 +1179,7 @@ Deno.test("describe()", async (t) => {
     async function assertAllOptions(
       cb: (fns: readonly [Spy, Spy]) => void,
     ) {
-      await assertOptions({ ...baseOptions, ignore: true }, cb);
+      await assertIgnoreOptions({ ...baseOptions, ignore: true }, cb);
     }
 
     await t.step("signature 1", async (t) => {

